### ISSUE TITLE
Package Popcorn Time for Debian

### ISSUE THREAD

**Issue by nodiscc** [popcorntime/popcorntime#143]
```
Operating System Version:
Debian Testing/Sid

Expected Behaviour:
The application should be installable through the OS package management system.

Actual Behaviour:
The application requires manual download, manual archive extraction, manually running an executable binary file.
```

Hi, it would be great if this project could provide .deb packages suitable for installation in Debian and derivatives.

There was a Request For Packaging filed at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=741972 for a previous fork. If a proper packaging procedure is found, distribution/repo maintainers will be able to provide package-based, automatic upgrades for the software (PPAs or inclusion in official repositories).

Thank you for your work on this great project.

**Comment by aurimus** [comment]
It used to work with a bash script installation, it still work on the other version: https://github.com/PopcornTimeCommunity/desktop/releases

**Comment by ChrisAlderson** [comment]
Fixed with #223
